manifests/m/MaxDiesel/UnrealCommander/3.5.7.1497/MaxDiesel.UnrealCommander.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MaxDiesel.UnrealCommander
+PackageVersion: 3.5.7.1497
+PackageLocale: en-US
+Publisher: Anton Diesel

```

```suggestion
Publisher: Max Diesel
```
This needs to match Control Panel.